build.gradle

```diff
@@ -359,3 +359,7 @@ task dist(dependsOn: assemble) {
 	group = 'Distribution'
 	description = 'Builds -dist, -docs distribution archives.'
 }
+
+wrapper {
+	gradleVersion = '4.9'
```

Comment:
We don't need this.
What we do is like:
1. we go into the `/gradle/wrapper` directory
2. open `gradle-wrapper.properties`
3. Modify the version in the `distributionUrl` property
4. Call `gradlew wrapper` task

That's all. No need in anything else. Also this way we keep Gradle version only in a single place.